ci/lint/04_install.sh

```diff
@@ -12,10 +12,11 @@ pushd "/"
 
 ${CI_RETRY_EXE} apt-get update
 # Lint dependencies:
+# - cargo (used to run the lint tests)
 # - curl/xz-utils (to install shellcheck)
 # - git (used in many lint scripts)
 # - gpg (used by verify-commits)
-${CI_RETRY_EXE} apt-get install -y curl xz-utils git gpg
+${CI_RETRY_EXE} apt-get install -y cargo curl xz-utils git gpg
```

Comment:
later we're reinstalling `curl` and `xz-utils`, we could remove them from there

Reply:
That's what I meant, of course, but if you think duplicating them is better for documentation, I'm fine with it of course